core/mondoo-azure-security.mql.yaml

```diff
@@ -52,7 +63,297 @@ policies:
+    title: Ensure all users in administrative roles have MFA enabled 
+    severity: 100
+    docs:
+      desc: |
+        Enable multifactor authentication for all users in administrative roles!
+      audit: |
+        __cnquery run__
+
+        To audit Microsoft Azure with `cnquery run`:
+
+        Run the following query:
+
+          ```bash
+          cnquery run azure -c "microsoft.security.latestSecureScores.controlScores.one( _['controlName'] == 'AdminMFAV2' && _['score'] == 10)" --certificate-path <*.pem> --tenant-id <tenant_id> --client-id <client_id>
+          ```
+
+        __cnquery shell__
+
+        To audit Microsoft Azure with `cnquery shell`:
+
+        1. Launch `cnquery shell`:
+
+          ```bash
+          cnquery shell azure --certificate-path <*.pem> --tenant-id <tenant_id> --client-id <client_id>
+          ```
+        
+        2. Run the following query:
+
+          ```mql
+          microsoft.security.latestSecureScores.controlScores.one( _['controlName'] == 'AdminMFAV2' && _['score'] == 10 )
+          ```
+      remediation: |
+        ### Microsoft Azure Portal
+
+        To update via the Microsoft Azure portal:
+        1. Log in as a Conditional Access Administrator, Security Administrator, or Global Administrator to the Microsoft Azure portal at https://portal.azure.com 
+        2. Browse to Azure Active Directory --> Security --> Conditional Access
+        3. Select New Policy --> with following conditions:
+           * Grant --> Grant access (Require multifactor authentication)
+           * Users --> Users and groups --> Include --> Select users and groups --> Directory roles --> at minimum select following rules (Billing admin, Conditional Access admin, Exchange admin, Global admin, Helpdesk admin, Security admin, SharePoint admin, and User admin)
+           * Cloud apps or actions --> All cloud apps 
+        5. Make sure that the policy is Enabled. You can also set Enable policy to Report-only, and After confirming your settings using report-only mode, an administrator can move the Enable policy toggle from Report-only to On. 
+    query: |
+      microsoft.security.latestSecureScores.controlScores.one( _['controlName'] == 'AdminMFAV2' && _['score'] == 10 )
+  - uid: mondoo-azure-security-ensure-that-between-two-and-four-global-admins-are-designated
+    title: Ensure that at least three global admins are designated and not more than four
+    severity: 60
+    docs:
+      desc: |
+        This control checks that there are enough Global Admins in a single tenant. 
+        When it comes to designating global admins, it's important to consider the size and complexity of the organization, as well as the level of responsibility and authority required for the role. As a general rule, it's a good idea to have at least three global admins to ensure that there is redundancy and coverage in case one admin is unavailable or leaves the organization.
```

Comment:
```suggestion
        When designating global admins, it's important to consider the size and complexity of the organization, as well as the level of responsibility and authority required for the role. As a general rule, it's a good idea to have at least three global admins to ensure that there is redundancy and coverage in case one admin is unavailable or leaves the organization.
```